packages/stitch/tests/remoteExecutors.test.ts

```diff
@@ -90,7 +90,11 @@ describe('remote executors and subscribers', () => {
       }]
     });
 
-    await graphql(stitchedSchema, `subscription { newPost { message } }`);
+    try {
+      await subscribe(stitchedSchema, parse(`subscription { newPost { message } }`));
```

Comment:
wait... wut? So that means GraphQL Express doesn't bother to review what method it needs to call... (?!)

https://github.com/graphql/express-graphql/blob/ed00ba731feec6f505f06916a43469db6eb9e1d5/src/index.ts#L233

WELP.

Reply:
good servers are so hard to find these days when trying to boycott Apollo.

Reply:
https://github.com/graphql/express-graphql/pull/687#issuecomment-736104093

Reply:
Yeah, I saw that. Super exciting that that's being incorporated after 6 months in PR.

Reply:
and, ironic timing...